sovrin_client/client/client.py

```diff
@@ -121,7 +121,7 @@ def postReplyRecvd(self, identifier, reqId, frm, result, numReplies):
                     # a malformed message should not result in an error message
                     # being shown on the cli since the clients would anyway
                     # collect enough replies from other nodes.
-                    logger.error("Observer threw an exception", exc_info=ex)
```

Comment:
@rajeshkalaria80 I could see that we are throwing an exception explicitly in client as Runtime exception. Apart from that we are not throwing it anywhere else. Also, we don't want to show it on UI and just want to add it to logs, that's what product team wants in this story. But I agree with your point, how would we know if something is not right in background. This was explicitly put in place due to this error being thrown too many time during a demo, and still everything was working fine. Only one or two nodes was throwing the exception and all other nodes returned correct results.